Update InventoryType.cs

TALESIOFIFREAK opened the pull request:
[Armor("Power Weight")] en,
[Armor("Magic Weight")] en2,
[Armor("Master Belt")] en3,

[Accessory("Breakthrough")] Item6f,
[Accessory("Cristal Regalia+")] Item70,

TALESIOFIFREAK committed:
Update InventoryType.cs

[Armor("Power Weight")] en,
[Armor("Magic Weight")] en2,
[Armor("Master Belt")] en3,

[Accessory("Breakthrough")] Item6f,
[Accessory("Cristal Regalia+")] Item70,

Mirrorman95 commented:
I'd like to point out that because these additions aren't updated in ArmorType.cs or AccessoryType.cs as well, their names are still placeholders in the Character Equipment editor menus.

TALESIOFIFREAK committed:
Update InventoryType.cs

[Armor("Power Weight")] en,
[Armor("Magic Weight")] en2,
[Armor("Master Belt")] en3,

[Accessory("Breakthrough")] Item6f,
[Accessory("Crystal Regalia+")] Item70,

TALESIOFIFREAK commented:
i changed this again because i has made the same mistake.
